docs/io/fits/index.rst

```diff
@@ -473,6 +473,42 @@ table's structure needs more information. First of all, tables can only be an
 extension HDU, not a primary. There are two kinds of FITS table extensions:
 ASCII and binary. We'll use binary table examples here.
 
+.. note::
+    
+    If you want to create a simple binary FITS table with no other HDUs,
+    you should use the Table_ interface in Astropy instead and then 
+    write to FITS.
+
+    >>> from astropy.table import Table
+    >>> t = Table([[1,2],[4,5],[7,8]], names=('a','b','c'))
+    >>> t.write('table.fits', format='fits')
+
+    Same code in PyFITS would look like this:
```

Comment:
Now that you mention it, PyFITS move to `astropy.io.fits` is so long ago now that we can probably replace PyFITS reference with `astropy.io.fits`. :thinking: